frontend/package.json

```diff
@@ -38,27 +40,27 @@
     "opentok-layout-js": "^5.4.0",
     "opentok-solutions-logging": "^1.1.5",
     "postcss": "^8.4.38",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
+    "react": "^19.1.0",
+    "react-dom": "^19.1.0",
     "react-router-dom": "^6.11.0",
     "resize-observer-polyfill": "^1.5.1",
     "tailwindcss": "^3.4.14",
     "typedoc": "^0.26.10",
     "ua-parser-js": "^1.0.37",
     "uuid": "^9.0.1",
-    "@vitejs/plugin-react": "^4.2.0",
     "vite": "^5.4.6",
     "vitest": "^1.6"
   },
   "devDependencies": {
     "@types/lodash": "^4.17.1",
-    "@types/react": "^18.3.3",
-    "@types/react-dom": "^18.2.1",
+    "@types/react": "^19.1.0",
+    "@types/react-dom": "^19.1.0",
     "@types/react-router-dom": "^5.3.3",
     "@types/ua-parser-js": "^0.7.39",
     "@types/uuid": "^9.0.8",
     "@vitest/coverage-v8": "^1.3.0",
     "@vitest/ui": "^1.3",
+    "babel-plugin-react-compiler": "19.1.0-rc.1",
```

Comment:
Note: please do not be scared that the name of the React Compiler starts with `babel-plugin`. While it works in babel, it is also fully supported with `vite` - as you can see, no `babel.config` files have been introduced and it is all handled within `vite.config.ts`